lte/cloud/go/services/subscriberdb/obsidian/handlers/handlers.go

```diff
@@ -89,11 +95,23 @@ var (
 	// Mobilityd states are keyed as <IMSI>.<APN>.
 	mobilitydStateKeyRe = regexp.MustCompile(`^(?P<imsi>IMSI\d+)\..+$`)
 
-	subscriberLoadCriteria       = configurator.EntityLoadCriteria{LoadMetadata: true, LoadConfig: true, LoadAssocsFromThis: true}
 	apnPolicyProfileLoadCriteria = configurator.EntityLoadCriteria{LoadAssocsFromThis: true, LoadAssocsToThis: true}
 )
 
-var subscriberStateTypes = []string{
+var subscriberStateTypesKeyedByIMSI = []string{
```

Comment:
This and the below var names at first look like they're map definitions. Took me 20s to realize. Can you add a short comment to each indicating why they're listed separately / what it means to be keyed this way